doc/userguide/rules/tls-keywords.rst

```diff
@@ -17,9 +17,21 @@ Examples::
 
 ``tls.cert_subject`` can be used as ``fast_pattern``.
 
-``tls.cert_subject`` replaces the previous keyword name: ``tls_cert_subject``. You may continue
-to use the previous name, but it's recommended that rules be converted to use
-the new name.
+tls.subject
+~~~~~~~~~~~
+
+Legacy keyword to match TLS/SSL certificate Subject field.
+
+example:
+
+::
+
+  tls.subject:"CN=*.googleusercontent.com"
+
+Case sensitive, can't use 'nocase', or other modifiers.
+
+**Note:** ``tls.cert_subject`` replaces the following keyword names: ``tls_cert_subject`` and ``tls.subject``. 
```

Comment:
Seems that we missed one review point here: https://github.com/OISF/suricata/pull/8461#discussion_r1087901976
(bringing this up because I do see it as a rewording that makes things more clear and is more in consonance with our documentation)

This applies to the other note, too.